src/modules/45-dashboards/components/AidaDrawer/__tests__/AidaDrawer.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { act, fireEvent, render, RenderResult, screen } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import AidaDrawer, { AidaDrawerProps } from '..'
+
+const renderComponent = (props: AidaDrawerProps): RenderResult =>
+  render(
+    <TestWrapper>
+      <AidaDrawer {...props} />
+    </TestWrapper>
+  )
```

Comment:
It can be super-handy to set some sensible defaults and then override only what you need in each test
```suggestion
const renderComponent = (props: Partial<AidaDrawerProps> = {}): RenderResult =>
  render(
    <TestWrapper>
      <AidaDrawer isOpen setIsOpen={jest.fn()} {...props} />
    </TestWrapper>
  )
```

Then later you can add only what you need to override. e.g.
```
  test('it should not display content within drawer when closed', async () => {
    renderComponent({ isOpen: false })